app/views/katello/api/v2/systems/index.json.rabl

```diff
@@ -1,3 +1,13 @@
-object false
+collection @collection
 
-extends "katello/api/v2/common/index"
+extends "katello/api/v2/common/metadata"
+
+child @collection[:results] => :results do
+  attributes :id => :katello_id
+  attributes :uuid, :name, :description
+  attributes :location
+  attributes :content_view, :content_view_id
+  attributes :distribution
+  attributes :katello_agent_installed? => :katello_agent_installed
+  attributes :registered_by
+end
```

Comment:
we've used a paradigm of base.json.rabl to define 'common' attributes, and then show and index simply extending that.  See content_view_version rabl templates for an example.  I'd like to see that here